src/lqfit.jl

```diff
@@ -0,0 +1,334 @@
+# lq compton region peakshapes
+f_lq_compton(x, v) = aoe_compton_peakshape(x, v.μ, v.σ, v.n, v.B, v.δ)
+f_lq_sig(x, v)     = aoe_compton_signal_peakshape(x, v.μ, v.σ, v.n)
+f_lq_bkg(x, v)     = aoe_compton_background_peakshape(x, v.μ, v.σ, v.B, v.δ)
+
+#######
+f_lq_mu(x, p) = p[1] .+ p[2].*x
+@. f_lq_sigma(x, p) = sqrt(abs(p[1]) + abs(p[2])/x^2)
+#######
+
+# lq compton centroids energy depencence
+MaybeWithEnergyUnits = Union{Real, Unitful.Energy{<:Real}}
+f_lq_μ(x::T, v::Array{<:T}) where T<:Unitful.RealOrRealQuantity = -v[1] * x + v[2]
+f_lq_μ(x::Array{<:T}, v::Array{<:T}) where T<:Unitful.RealOrRealQuantity = f_lq_μ.(x, v)
+f_lq_μ(x, v::NamedTuple) = f_lq_μ(x, [v.μ_scs_slope, v.μ_scs_intercept])
+
+# lq compton sigma energy depencence
+f_lq_σ(x::T, v::Array{<:T}) where T<:Unitful.RealOrRealQuantity = exponential_decay(x, v[1], v[2], v[3])
+f_lq_σ(x::Array{<:T}, v::Array{<:T}) where T<:Unitful.RealOrRealQuantity = exponential_decay.(x, v[1], v[2], v[3])
+f_lq_σ(x, v::NamedTuple) = f_lq_σ(x, [v.σ_scs_amplitude, v.σ_scs_decay, v.σ_scs_offset])
+
+"""
+    estimate_single_peak_stats_psd(h::Histogram{T}) where T<:Real
+
+Estimate peak parameters for a single peak in a histogram using the maximum, the FWHM and the area of the peak.
+
+# Returns
+    * `peak_pos`: Position of the peak
+    * `peak_fwhm`: Full width at half maximum of the peak
+    * `peak_sigma`: Standard deviation of the peak
+    * `peak_counts`: Counts of the peak
+    * `mean_background`: Mean background of the peak
+"""
+function estimate_single_peak_stats_psd_lq(h::Histogram{T}) where T<:Real
```

Comment:
Is this any differen to the `A/E` code? if not, just call the code for that here.
``` julia
function estimate_single_peak_stats_psd_lq(h::Histogram{T}) where T<:Real             estimate_single_peak_stats_psd(h)
end
```